tests/contract/test_resource_client.py

```diff
@@ -1303,6 +1303,12 @@ def test_assert_write_only_property_does_not_exist_fail(resource_client, schema)
         resource_client.assert_write_only_property_does_not_exist(created_resource)
 
 
+def test_get_value_by_key_path_with_string_key(resource_client):
+    model = {"a": 1, "b": 2}
```

Comment:
shouldn't this be tested with a nested property?

Reply:
This test case if for the case where the key path is just a string. I can create another test case for nested key path.